2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 2 Line 10_
```markdown
### Fundamentals
There are several fundamental types of objects in python:
* `bool`: with values True and False
* `str`: for alphanumeric text like "Hello world"
* `int`: for integers like 1, 42, and -5
* `float`: for floating point numbers like 96.8

### Containers 
```
Fundamentals vs containers is a nice taxonomy to break these down